packages/react-components/src/DocsComponents/FluentDocsDocsStory.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import { Anchor, Description, DocsStoryProps, Story, Subheading } from '@storybook/addon-docs';
+import * as React from 'react';
+import {
+  githubLightTheme,
+  SandpackCodeEditor,
+  SandpackPreview,
+  SandpackProvider,
+  SandpackThemeProvider,
+  useSandpack,
+} from '@codesandbox/sandpack-react';
+import '@codesandbox/sandpack-react/dist/index.css';
+import * as dedent from 'dedent';
+import newGithubIssueUrl from 'new-github-issue-url';
+import * as packageJson from '../../package.json';
+import bugReportTemplate from '../../../../.github/ISSUE_TEMPLATE/bug_report_v9.md';
+
+const handleReportBug = (componentName: string) => async (e: React.MouseEvent) => {
+  // create codesandbox
+  let codesandboxUrl = '';
+  const openInCSButton = e.currentTarget.nextElementSibling;
+  const codesandboxForm = openInCSButton?.getElementsByTagName('form').item(0);
+  if (codesandboxForm) {
+    const codesandboxFormData = new FormData(codesandboxForm);
+    codesandboxFormData.append('json', '1');
+    const codesandboxResponse = await fetch(codesandboxForm.action, {
+      method: 'post',
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Can this be avoided?

Reply:
I've tried, but couldn't find any easy way to avoid it